GOVERNANCE.md

```diff
@@ -0,0 +1,216 @@
+>   the consent of all participants, though allowing for some dissent
+>   (rough consensus), and to have the actual products of engineering
+>   (running code) trump theoretical designs.
+>
+>   Having full consensus, or unanimity, would be ideal, but we don't
+>   require it: Requiring full consensus allows a single intransigent
+>   person who simply keeps saying "No!" to stop the process cold.  We
+>   only require rough consensus: If the chair of a working group
+>   determines that a technical issue brought forward by an objector has
+>   been truly considered by the working group, and the working group has
+>   made an informed decision that the objection has been answered or is
+>   not enough of a technical problem to prevent moving forward, the
+>   chair can declare that there is rough consensus to go forward, the
+>   objection notwithstanding.
+
+Within the RIOT community, the duties of an IETF working group chair fall to maintainers knowledgeable in the area of expertise.
+This knowledgability is determined by their own contributions.
+On decisions regarding a release, the release manager(s) take this position.
+
+## Meetings
+
+There are 3 types of meetings that maintainers may participate in:
+
+- The annual General Assembly (GA),
+- The quarterly Virtual Maintainer Assembly (VMA), and
+- and the Weekly Coordinational Meeting.
+
+The GA is public and anyone who sees themselves as a member of the RIOT community can participate.
+Larger community steering decisions for the community are made during the GA, e.g., electing contact people for the Code of Conduct.
+The GA usually takes place during the [RIOT Summit](https://summit.riot-os.org/), the annual get-together of the RIOT community.
+The GA moderator usually is appointed by the organizers of the RIOT Summit.
+It is usually recorded and its notes will be published publicly in the RIOT forum.
+The agenda for the GA is collected before the assembly but may be bashed at the start of the meeting.
+
+The Virtual Maintainer Assembly (VMA) is a closed meeting among maintainers.
+The VMA appoints the release manager for upcoming releases and the moderator for the next VMA.
+Other maintenance decisions such as the fate of larger sections of code are discussed after these administrative tasks are done.
+The VMA usually takes place about a month after the latest release, usually in a virtual space, such as a video conference, the VMA moderator polls the maintainers for a sufficient date around the date of the upcoming release.
+The VMA may or may not co-incide with the GA, usually the VMA moderator decides, however, to merge these two meetings.
+In this case, VMA moderator and RIOT Summit organizers decide together on who is moderating the joint event (it may be the VMA moderator, it may be someone else).
+The agenda for the VMA is collected before the assembly but may be bashed at the start of the meeting.
+The notes of the VMA will be published publicly in the RIOT forum.
```

Comment:
```suggestion
The notes of the VMA will be published publicly in the [RIOT forum](https://forum.riot-os.org/tag/vma).
```

Reply:
I intentionally try to avoid links to platforms beyond our website or GitHub in this document, as those might change in the future, while putting them in here would just ossify them (or the links could spoil like milk,... see the old maintainer list or other links [I just had to move around](https://github.com/RIOT-OS/RIOT/pull/21066)), see also https://github.com/RIOT-OS/RIOT/pull/21067#discussion_r1878063875.

Reply:
to avoid spoiling we could link a reference document and links to people lists (maintainer, moderator-list .. and covers link like this)

Reply:
We have already the README and many other documents (CONTRIBUTING.md to name one) that have such lists. Do we really need yet another one?

Reply:
right, i thought about the other link in this document but avoiding more links is probably better.